### PR TITLE
chore(core): enable wandb-core by default

### PR DESCRIPTION
**Pull request by kptkin**
!!!! DONT MERGE UNTIL WE ARE READY FOR RELEASE  0.18.0 !!!!

Description
-----------

What does the PR do? Include a concise description of the PR contents.

- [ ] I updated CHANGELOG.md, or it's not applicable

Testing
-------
How was this PR tested?

### CODE REVIEW

File: core/pkg/server/sender.go
Comment (timoffex) [inline core/pkg/server/sender.go:1]: this can probably go in earlier in a separate PR

**Comment by timoffex** [comment]
I think it would make sense to fix tests in a separate PR as well to keep this one as simple as possible, since it's so important.

**Comment by kptkin** [comment]
> I think it would make sense to fix tests in a separate PR as well to keep this one as simple as possible, since it's so important.

yeah, i have a few PRs that i will merge before this one
